setup.py

```diff
@@ -110,4 +109,6 @@ def read(*rnames):
     ],
     include_package_data=True,
     zip_safe=False,
+    # XXX: This doesn't pick up the doctests.
+    test_suite='zope.file.tests',
```

Comment:
Easily fixable by defining an `additonal_tests()` function in `zope.file.tests` that returns the doctests.

The zope.testrunner `test_suite()` can then return `unittest.TestSuite([additional_tests(), unittest.defaultTestLoader.loadTestsFromName(__name__)])` to avoid having to list the doctest files twice.  Here's an example: https://github.com/gtimelog/gtimelog/blob/master/src/gtimelog/tests/test_timelog.py#L1558-L1567

OTOH `python setup.py test` is a disappointment, so I'm not sure it's worth the trouble to support it correctly.